x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleInitialisationTests.java

```diff
@@ -364,6 +368,38 @@ public void testMasterFailover() throws Exception {
         });
     }
 
+    public void testCreatePolicyOnClose() throws Exception {
+        // start master node
+        logger.info("Starting server1");
+        final String server_1 = internalCluster().startNode();
+        final String node1 = getLocalNodeId(server_1);
+
+        assertAcked(client().execute(StopILMAction.INSTANCE, new StopILMRequest()).get());
+        GetStatusAction.Response statusResponse = client().execute(GetStatusAction.INSTANCE, new GetStatusAction.Request()).get();
+        assertThat(statusResponse.getMode(), equalTo(OperationMode.STOPPING));
```

Comment:
Should this wait for it to be fully stopped rather than just stopping?

Reply:
yeah, I kind of rushed this part, but I will come back to it. I think this requires the 
runner to be triggered somehow, right?

Reply:
if so, might be another cleanup thingy to check on

Reply:
followed up. looks like waiting is all that is needed. for some reason I thought otherwise